test/page.test.ts

```diff
@@ -110,5 +113,132 @@ describe('page', () => {
         font,
       });
     });
+
+    it('renders rect objects', () => {
+      const frame: Frame = {
+        ...{ x: 10, y: 20, width: 200, height: 30 },
+        objects: [{ type: 'rect', x: 1, y: 2, width: 3, height: 4 }],
+      };
+
+      renderFrame(frame, page);
+
+      expect(pdfPage.drawRectangle).toHaveBeenCalledWith({
+        x: 11, // 10 + 1
+        y: 774, // 800 - 20 - 2 - 4
```

Comment:
Maybe replace `11` by `10+1` (and analogically for 774)?

Reply:
:+1: Good point! :heavy_check_mark: